tests/unit/mock_data/all.workflow.database.tsv

```diff
@@ -0,0 +1,6 @@
+DATE	CROMWELL_SERVER	RUN_ID	WDL_NAME	STATUS	ALIAS
+20200110_105202	https://cromwell-v47.dsde-methods.broadinstitute.org	a63aa10c-a43e-4ca7-9be9-c2d2aa08b96d	ValidateBam.wdl Failed
```

Comment:
I don't see any of the runs being "aliased", so is not representative of real world scenarios.
It would be better if you have aliases for some of the runs, but not all; this facilitates later tests when aliases are allowed.

Reply:
done